### PR TITLE
feature/FOUR-13870: FOUR-13382 Launchpad Setup - Create a dropdown to show display screen

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-13870

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

### CODE REVIEW

File: resources/js/components/shared/LaunchpadSettingsModal.vue [hunk 1]
```diff
@@ -127,6 +127,34 @@
...
+          <button
+            id="statusDropdownScreen"
+            class="btn dropdown-toggle dropdown-style w-100 d-flex justify-content-between align-items-center btn-custom"
+            type="button"
+            data-toggle="dropdown"
+            aria-haspopup="true"
+            aria-expanded="false"
+          >
+            <div class="d-flex align-items-center">
+              <span class="ml-2 custom-text">{{ selectedScreen || 'Select Screen' }}</span>
```
Comment (fagubla) [inline resources/js/components/shared/LaunchpadSettingsModal.vue:141]: missing translation

File: resources/js/components/shared/LaunchpadSettingsModal.vue [hunk 2]
```diff
@@ -127,6 +127,34 @@
...
+            class="dropdown-menu custom-dropdown"
+            aria-labelledby="statusDropdownScreen"
+          >
+            <a
+              v-for="(item, index) in dropdownSavedScreen"
+              :key="index"
+              class="dropdown-item"
+              @click="selectScreenOption(item)"
+            >
+              {{ item.title || 'Select Screen' }}
```
Comment (fagubla) [inline resources/js/components/shared/LaunchpadSettingsModal.vue:154]: missing translation

File: resources/js/components/shared/LaunchpadSettingsModal.vue [hunk 3]
```diff
@@ -443,6 +493,33 @@ export default {
           this.dropdownSavedCharts = [];
         });
     },
+    /**
+     * Initial method to retrieve Saved Search Charts and populate dropdown
+     * Package Collections and Package SavedSearch always go together
+     */
+     retrieveDisplayScreen() {
+      if (!ProcessMaker.packages.includes("package-collections")) return;
```
Comment (fagubla) [inline resources/js/components/shared/LaunchpadSettingsModal.vue:501]: why?

File: resources/js/components/shared/LaunchpadSettingsModal.vue [hunk 4]
```diff
@@ -443,6 +493,33 @@ export default {
           this.dropdownSavedCharts = [];
         });
     },
+    /**
+     * Initial method to retrieve Saved Search Charts and populate dropdown
+     * Package Collections and Package SavedSearch always go together
```
Comment (fagubla) [inline resources/js/components/shared/LaunchpadSettingsModal.vue:498]: ???

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by pmPaulis** [comment]
https://ci-32d77c25dd.engk8s.processmaker.net/

**Pull request merged**